crates/codegen/language/definition/src/compiler/analysis/mod.rs

```diff
@@ -75,4 +75,8 @@ impl SpannedLanguage {
             .flat_map(|section| &section.topics)
             .flat_map(|topic| &topic.items);
     }
+
+    fn built_ins(&self) -> impl Iterator<Item = &SpannedBuiltIn> {
```

Comment:
nit: I wonder if this is still needed? `self.built_ins` is already a public iterable, and the single callsite can consume it the same way.

Reply:
It's not needed, I wanted to keep uniform wrt. `items()`. But I agree it doesn't make much sense. I'll remove it.